psydac/mapping/discrete_gallery.py

```diff
@@ -15,9 +15,9 @@
 
 class Collela3D( Mapping ):
 
-    expressions = {'x':'2.*(x1 + 0.1**sin(2.*pi*x1)*sin(2.*pi*x2)) - 1.',
-                   'y':'2.*(x2 + 0.1**sin(2.*pi*x1)*sin(2.*pi*x2)) - 1.',
-                   'z':'2.*x3  - 1.'}
+    _expressions = {'x':'2.*(x1 + 0.1*sin(2.*pi*x1)*sin(2.*pi*x2)) - 1.',
+                    'y':'2.*(x2 + 0.1*sin(2.*pi*x1)*sin(2.*pi*x2)) - 1.',
+                    'z':'2.*x3  - 1.'}
```

Comment:
Clearly we were not using this class before.
Should we not add it to `sympde/topology/analytical_mapping.py`?